src/helpers/userHelper.js

```diff
@@ -1482,6 +1485,131 @@ const userHelper = function () {
       logger.logException(err);
     }
   };
+
+  const executeSummaryGroupHelper = async function () {
+    try {
+      const validSummaryGroups = await summaryGroup.find(
+        {
+
+          isActive: true,
+          summaryReceivers: { $ne: [] },
+          teamMembers: { $ne: [] },
+        },
+        '_id summaryGroupName summaryReceivers teamMembers',
+      );
+
+      for (let i = 0; i < validSummaryGroups.length; i++) {
+        const validSummaryGroup = validSummaryGroups[i];
+        const { summaryGroupName } = validSummaryGroup;
+        const { summaryReceivers } = validSummaryGroup;
+
+        const summaryReceiverEmails = summaryReceivers.map(receiver => receiver.email);
+        console.log('summaryReceiverEmails', summaryReceiverEmails);
+
+        const { teamMembers } = validSummaryGroup;
+
+        const emails = [];
+        let emailBody = `<h2>Weekly Summaries for ${summaryGroupName}:</h2>`;
+        const weeklySummaryNotProvidedMessage = '<div><b>Weekly Summary:</b> <span style="color: red;"> Not provided! </span> </div>';
+
+        const weeklySummaryNotRequiredMessage = '<div><b>Weekly Summary:</b> <span style="color: magenta;"> Not required for this user </span></div>';
+        for (let i = 0; i < teamMembers.length; i += 1) {
+          const teamMember = teamMembers[i];
+          const userId = mongoose.Types.ObjectId(teamMember._id);
+          const { fullName } = teamMember;
+
+          const userProfileDocument = await userProfile.findById(userId).select({
+            weeklySummaries: 1,
+            weeklycommittedHours: 1,
+            mediaUrl: 1,
+            weeklySummariesCount: 1,
+          });
+
+
+          const {
+            weeklySummaries,
+            weeklycommittedHours,
+            mediaUrl,
+            weeklySummariesCount,
+          } = userProfileDocument;
+
+
+          // weeklySummaries array will have only one item fetched (if present),
+          // consequently totalSeconds array will also have only one item in the array (if present)
+          // hence totalSeconds[0] should be used
+          const hoursLogged = weeklycommittedHours && weeklycommittedHours.totalSeconds
+              ? weeklycommittedHours.totalSeconds[0] / 3600
```

Comment:
```suggestion
          const hoursLogged = lastWeekTangibleHrs && lastWeekTangibleHrs.totalSeconds
              ? lastWeekTangibleHrs.totalSeconds[0] / 3600
```
I think these should be lastWeekTangibleHrs instead of WeeklyCommittedHours?